lib/signs/utilities/source_config.ex

```diff
@@ -214,4 +214,8 @@ defmodule Signs.Utilities.SourceConfig do
   def sign_routes({s}) do
     Enum.flat_map(s, &(&1.routes || []))
   end
+
+  def default_headway_group([headway_group, _]), do: headway_group
```

Comment:
What about completely abstracting access to this field with a helper like `sign_headway_group(sign, line \\ :top)`? Then we could use it in all three places.

Reply:
Good call! This did simplify things a lot and I think we can subsequently reduce the number of places we call the function to just 2. Just pushed the changes!